apps/assessment_registry/tests/test_dashboard_schema.py

```diff
@@ -185,7 +185,7 @@ def _query_check(minput, **kwargs):
             limitations="test",
             objectives="test",
             noOfPages=10,
-            publicationDate="2023-01-01",
+            publicationDate=str(date.today()),
```

Comment:
Let's use static values for testing.

Change the fixture used for the publicateDate to use a static value.